tensorflow/python/kernel_tests/cholesky_op_test.py

```diff
@@ -87,6 +88,26 @@ def testEmpty(self):
     self._verifyCholesky(np.empty([0, 2, 2]))
     self._verifyCholesky(np.empty([2, 0, 0]))
 
+class CholeskyGradTest(tf.test.TestCase):
+
+  def testCholeskyGrad(self):
+    np.random.seed(0)
+    backprop_block_size = 32    
+    shapes = ( (elem, 2*elem) for elem in [1,2,10,backprop_block_size+1,2*backprop_block_size+1] )
+    dtypes = (tf.float32, tf.float64)
+    with self.test_session(use_gpu=False):
+      for shape in shapes:             
+        for dtype in dtypes:
+          x = tf.constant( np.random.rand(shape[0]) )
+          R = tf.constant( np.random.randn(shape[0],shape[1])  )
```

Comment:
Unfortunately we don't run it all that much on the python code (the clang-format one is more or less enforced internally), but you might want to try pyformat --yapf (which is what I use, with two space indent rather than 4).

https://github.com/google/yapf